meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -90,16 +104,31 @@ deploy() {
 # MAIN
 #
 
-export DOCKER_HOST="unix:///var/run/docker-host.sock"
-
-if [ -z "$container_id" ]; then
-	echo "[ERROR] Container ID provided as an argument is needed."
-	exit 1
-fi
+# Do a dry run for copying the boot files and figure out if we would get in an
+# out of space situation
+boot_space="$(df -B1 --output=avail $boot_mountpoint | grep -v Avail)"
+available="$boot_space"
+available_threshold="524288" # All sizes in bytes
+printf "[INFO] Checking if boot partition can accommodate the new update... "
+for filename in $(find /resin-boot -type f | sed 's#/resin-boot##g'); do
+	if isBlacklisted "$filename"; then
+		continue
+	fi
+	filesize="$(du -b "/resin-boot${filename}" | cut -f1)"
```

Comment:
This will print the disk usage of that file that can be misleading. For example here it will measure the ext4 metadata and on a compressed filesystem it will report the compressed size. Consider switching to:

```bash
filesize=$(stat --format %s "$file")
```

Reply:
Done. Will push with next iteration.